.github/workflows/tests.yaml

```diff
@@ -0,0 +1,21 @@
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          cache: 'pip'
+      - name: Install test dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r tests/requirements.txt
+      - name: Test with pytest
+        run: |
+          python -m pytest tests/test_terraform.py --doctest-modules --junitxml=junit/test-results.xml --cov=com --cov-report=xml --cov-report=html
```

Comment:
What do all these args do - can we strip this to what we care about?
also probably either `tests/` or just no path at all

Reply:
The extra args were for code coverage metrics, but I guess we don't need them right now.

I updated the command to test the entire `tests/` directory